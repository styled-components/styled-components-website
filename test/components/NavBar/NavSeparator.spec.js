import React from 'react';
import renderer from 'react-test-renderer';
import NavSeparator from '../../../components/Nav/NavSeparator';

test('NavSeparator renders correctly', () => {
  const tree = renderer.create(<NavSeparator />).toJSON();

  expect(tree).toMatchSnapshot();
});
