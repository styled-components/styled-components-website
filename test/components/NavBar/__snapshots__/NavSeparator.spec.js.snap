// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavSeparator renders correctly 1`] = `
.c0 {
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  width: 0.2777777777777778rem;
  height: 0.2777777777777778rem;
  margin: 0 0.8333333333333334rem;
  border-radius: 50%;
  background: currentColor;
  opacity: 0.35;
}

<span
  className="c0"
/>
`;
